Jenkinsfile

```diff
@@ -170,6 +170,7 @@ def RunIntegrationTest(k, v) {
         script{
             try{
                 withEnv(["KUBE_VERSION=${v}"]){
+                    sh "tests/scripts/kubeadm.sh clean"
```

Comment:
nit: how about moving this down a couple lines so we can see the intended version of k8s in the log first

Reply:
If k8s isn't installed yet (already clean), is this a no-op?

Reply:
done. Nope clean doesn't have any effect even if there is no k8s cluster present